src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -156,6 +156,17 @@ internal class DefaultBuildScriptGenerator : IBuildScriptGenerator
                     }
 
                     platformVersion = detectionResult.LanguageVersion;
+                    _logger.LogDebug(
+                        "Using user supplied platform {platformName} and detected version {platformVersion}.",
```

Comment:
Do you mean detected platform?

Reply:
No, this is where user supplied the platform but not the version. We already have logging statements for auto-detection of a repo when explicit info was provided by the user